chapters/synchronization.txt

```diff
@@ -789,24 +789,12 @@ endif::VK_KHR_depth_stencil_resolve[]
     operation.
     Accesses of this type are not performed through a resource, but directly
     on memory.
-  * ename:VK_ACCESS_MEMORY_READ_BIT specifies read access via non-specific
-    entities.
-    These entities include the Vulkan device and host, but may: also include
-    entities external to the Vulkan device or otherwise not part of the core
-    Vulkan pipeline.
-    When included in a destination access mask, makes all available writes
-    visible to all future read accesses on entities known to the Vulkan
-    device.
-  * ename:VK_ACCESS_MEMORY_WRITE_BIT specifies write access via non-specific
-    entities.
-    These entities include the Vulkan device and host, but may: also include
-    entities external to the Vulkan device or otherwise not part of the core
-    Vulkan pipeline.
-    When included in a source access mask, all writes that are performed by
-    entities known to the Vulkan device are made available.
-    When included in a destination access mask, makes all available writes
-    visible to all future write accesses on entities known to the Vulkan
-    device.
+  * ename:VK_ACCESS_MEMORY_READ_BIT is equivalent to the logical OR of
+    every other etype:READ access flag bit that is supported in the
+    pipeline stage it is used with.
+  * ename:VK_ACCESS_MEMORY_WRITE_BIT is equivalent to the logical OR of
+    every other etype:WRITE access flag bit that is supported in the
```

Comment:
Probably doesn't matter, but I think the nomenclature usually used here is "bits being set" rather than "logical OR of values".

Also maybe should qualify "functionally equivalent" as the values are not actually equivalent.